app/api-v1/controllers/Order/helpers.js

```diff
@@ -51,7 +51,7 @@ const buildOrderOutput = (data, recipes) => ({
 exports.mapOrderData = async (data) => {
   if (!data.items || data.items.length < 1) throw new NothingToProcess()
   const records = await db.getRecipeByIDs(data.items)
-  const tokenIds = records.map((el) => el.latest_token_id)
+  const tokenIds = records.map((el) => el.token_id)
```

Comment:
I'm pretty sure the correct field is `latest_token_id` based on the migration

Reply:
so it was not my accidental changes! Yea. will update now